build.gradle

```
@@ -11,6 +11,11 @@ repositories {
 dependencies {
     testCompile('org.junit.jupiter:junit-jupiter:5.4.2')
     testCompile('org.assertj:assertj-core:3.11.1')
+    implementation 'junit:junit:4.12'
+    implementation 'org.junit.jupiter:junit-jupiter'
+    implementation 'org.junit.jupiter:junit-jupiter'
+    implementation 'org.junit.jupiter:junit-jupiter'
+    implementation 'org.junit.jupiter:junit-jupiter'
```

```suggestion
    implementation 'junit:junit:4.12'
    implementation 'org.junit.jupiter:junit-jupiter'
```

이건 뭔가 복사가 잘못 된 것으로 보이네요 ㅠㅠ